index copy.html

```diff
@@ -0,0 +1,385 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>The Portfolio of Massaillon Home page</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/8ea4cd9e84.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <header id="header-container">
+    <div class="logo-container">
+        <a href="#">Welcome</a>
+    </div>
+
+    <nav class="navbar">
+        <div class="underline-bar-navmenu hide"></div>
+        <ul class="nav-menu hide">
+            <li class="nav-item">
+                <a  href="#main-home" class="nav-link">Portfolio</a>
+            </li>
+            <li class="nav-item">
+                <a href="#About-Me" class="nav-link">About</a>
+            </li>
+            <li class="nav-item">
+                <a href="#form" class="nav-link" >Contact</a>
+            </li>
```

Comment:
- [ ] Kindly adjust the spacing as there is more spacing from the top.
- [ ] If you see the background blur then It is not like the Figma design so Kindly make the less blur to stick with Figma design.
### Figma design
![spac](https://user-images.githubusercontent.com/81727988/168334705-0d3979f8-b6c1-4a02-9da5-2820c2c82892.png)

### Your design

![adjust](https://user-images.githubusercontent.com/81727988/168334877-b84a05be-7680-4cbb-8672-eeb0ca5bb2c3.png)